rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
```

Comment:
- [ ] `person` and `book` should also have `Getters and Setters` both instead of just `Getters` as we are reading and also assigning the value to the variables.